root.sh

```
@@ -53,9 +52,16 @@ prefer_system_check: |
   done
 incremental_recipe: |
   make ${JOBS:+-j$JOBS} install
-  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+  mkdir -p "$INSTALLROOT/etc/modulefiles"
+  alibuild-generate-module --bin --lib > "$INSTALLROOT/etc/modulefiles/$PKGNAME"
+  cat >> "$INSTALLROOT/etc/modulefiles/$PKGNAME" <<\EoF
+  setenv ROOT_RELEASE $version
+  setenv ROOT_BASEDIR $::env(BASEDIR)/ROOT
+  setenv ROOTSYS $PKG_ROOT
+  prepend-path PYTHONPATH $PKG_ROOT/lib
+  EoF
```

Shellcheck (0.11.0)

[error] Remove indentation before end token (or use <<- and indent with tabs).

(SC1039)